music-journal/components/TopTracks.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { Container, Row, Card, CardColumns } from 'react-bootstrap/';
+import { getTopTracks } from '../utils/fetcher'
+
+const TopTracks = (props) => {
+    const [topTracks, setTopTracks] = useState()
+
+    useEffect(() => {
+        const initTopTracks = async () => {
+            try {
+                const response = await getTopTracks(props.token)
+                setTopTracks(response)
+            } catch (error) {
+                console.log('top tracks', error)
+            }
+        }
+        initTopTracks()
+    }, [])
+
+    return (
+        <Container>
```

Comment:
love how well this scales to different screen sizes 👏

Reply:
The internet provides: https://css-tricks.com/perfect-full-page-background-image/